src/widgets/frequencyGraph.js

```diff
@@ -0,0 +1,49 @@
+import { Op } from 'sequelize';
+import {
+  ActivityReport,
+  sequelize,
+} from '../models';
+import { REPORT_STATUSES, TOPICS, REASONS } from '../constants';
+
+async function countOccurrences(scopes, column, possibilities) {
```

Comment:
In this PR https://github.com/adhocteam/Head-Start-TTADP/pull/480/ I started a widgets/helper.js file. I would think it makes sense to move this function in there, and also maybe we could standardize on this and the topicFrequencyGraph endpoint.

Reply:
My loose plan is to replace the `topicFrequencyGraph` completely with this new widget. Need to raise the issue with patrice/angela when demoing this widget.